### PR TITLE
[MIG] 10.0 stock available unreserved

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
:+1: Code review

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM! :+1:

**Review by Cedric-Pigeon — APPROVED** [review]

File: stock_available_unreserved/models/product.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            tmpl.qty_available_stock_text = text
+
+    @api.multi
+    def _product_available(self, name=None, arg=False):
+        prod_available = super(ProductTemplate, self)._product_available(name,
+                                                                         arg)
+
+        variants = self.env['product.product']
+        for product in self:
+            variants += product.product_variant_ids
```
Comment (lmignon) [inline stock_available_unreserved/models/product.py]: @StefanRijnhart 
``` python

variants = self.mapped('product_variant_ids')
```

File: stock_available_unreserved/models/product.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            qty = res[prod.id]['qty_available_not_res']
+            text = res[prod.id]['qty_available_stock_text']
+            prod.qty_available_not_res = qty
+            prod.qty_available_stock_text = text
+
+    @api.model
+    def _prepare_domain_available_not_res(self, products):
+        domain_products = [('product_id', 'in', products.mapped('id'))]
+        domain_quant = []
+        domain_quant_loc, _, _ = products._get_domain_locations()
```
Comment (lmignon) [inline stock_available_unreserved/models/product.py]: @StefanRijnhart  Be carefull by using `_` as variable you hide  `from odoo import api, fields, models, _`

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by StefanRijnhart** [comment]
@lmignon comments attended to, thanks!

**Review by lmignon — APPROVED** [review]
Thank you! :+1:

**Pull request merged**